src/internal/index.ts

```diff
@@ -1 +1,42 @@
-export * from './internal'
+import { type CoverageModule, HANDLER } from './handler'
+
+export type JSONObject = { [x: string]: JSONValue | undefined }
+export type JSONArray = Array<JSONValue>
```

Comment:
so far in our .ts files we prefer `JSONValue[]` but it's yalls call on style in the guest product area, seems like the instrumentor you wrote uses `[]`

Reply:
Thanks, I'm willing to follow conventions in the NB. Do we also have a prettier config that the all .ts files follow?

Reply:
I'll chat you our eslint offline